src/sso/sso.spec.ts

```diff
@@ -144,7 +144,7 @@ describe('SSO', () => {
           });
 
           expect(url).toMatchInlineSnapshot(
-            `"https://api.workos.com/sso/authorize?client_id=proj_123&connection=connection_123&login_hint=foo%40workos.com&redirect_uri=example.com%2Fsso%2Fworkos%2Fcallback&response_type=code&state=custom%20state"`,
+            `"https://api.workos.com/sso/authorize?client_id=proj_123&connection=connection_123&login_hint=foo%40workos.com&redirect_uri=example.com%2Fsso%2Fworkos%2Fcallback&response_type=code&state=custom+state"`,
```

Comment:
`query-string` previously encoded spaces as `%20`, while `URLSearchParams` uses a `+`.

The WorkOS supports both, [though it seems like `+` is the preferred method](https://stackoverflow.com/questions/1211229/in-a-url-should-spaces-be-encoded-using-20-or).